pom.xml

```diff
@@ -213,8 +292,31 @@
             </plugin>
         </plugins>
     </build>
+
     <!-- Profiles -->
     <profiles>
+
+        <profile>
+            <id>dev</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
+            </activation>
+            <properties>
+                <build.profile.id>dev</build.profile.id>
+                <skip.integration.tests>true</skip.integration.tests>
+                <skip.unit.tests>false</skip.unit.tests>
+            </properties>
+        </profile>
+
+        <profile>
+            <id>integration-test</id>
```

Comment:
this profile has to be active when it runs within Travis, let's figure out who detect TravisCI environment :)